renga/api/repository.py

```diff
@@ -156,8 +157,16 @@ def init_repository(
             if self.git is None:
                 self.git = GitRepo.init(str(path))
         else:
-            assert self.git is None, 'Git repo already exists.'
-            self.git = GitRepo.init(str(path))
+            if self.git is not None:
+                # Cleanup
+                self.renga_path.rmdir()
+                raise QuestionableGitOperation(
+                    'Git repo already exists. Please use the `--force` flag ' +
```

Comment:
You can simply rely on Python parser to join static strings, hence please remove the `+`.